dags/veda_data_pipeline/veda_promotion_pipeline.py

```diff
@@ -18,30 +20,31 @@
 - This DAG can run with the following configuration <br>
 ```json
 {
-    "collection": "collection-id", 
-    "data_type": "cog", 
-    "description": "collection description", 
-    "discovery_items": 
+    "collection": "collection-id",
+    "data_type": "cog",
+    "description": "collection description",
+    "discovery_items":
         [
             {
-                "bucket": "veda-data-store-staging", 
-                "datetime_range": "year", 
-                "discovery": "s3", 
-                "filename_regex": "^(.*).tif$", 
+                "bucket": "veda-data-store-staging",
+                "datetime_range": "year",
+                "discovery": "s3",
+                "filename_regex": "^(.*).tif$",
                 "prefix": "example-prefix/"
             }
-        ], 
-    "is_periodic": true, 
-    "license": "collection-LICENSE", 
-    "time_density": "year", 
-    "title": "collection-title"
+        ],
+    "is_periodic": true,
+    "license": "collection-LICENSE",
+    "time_density": "year",
+    "title": "collection-title",
+    "transfer": "false"
 }
 ```
 """
 
 dag_args = {
     "start_date": pendulum.today("UTC").add(days=-1),
-    "schedule_interval": None,
+    "schedule": None,
```

Comment:
@ividito Is it okay to update this? `schedule_interval` is deprecated. https://github.com/apache/airflow/pull/25410

https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#deprecation-of-schedule-interval-and-timetable-arguments-25410